core/tm/routes/profesional.ts

```diff
@@ -475,23 +478,25 @@ router.post('/profesionales', Auth.authenticate(), (req, res, next) => {
 
     }
     if (req.body.profesional) {
-        // console.log(req.body.profesional);
-        // profesional.findOne({
-        //     'documento': req.body.profesional.documento
-        // }, (err, person) => {
-        //     if (person) {
-        //         res.json(null);
-        //     } else {
+
         const newProfesional = new profesional(req.body.profesional);
         Auth.audit(newProfesional, req);
-        newProfesional.save((err2) => {
+        newProfesional.save(async (err2) => {
+
+
+            try {
+                await log(req, 'profesional:post', null, 'profesional:post', newProfesional, null);
+            } catch (err) {
+                next(err);
```

Comment:
No debería ir el try o debería cubrir toda la función.
```suggestion
                return next(err);
```